pipeline/terraform/main.tf

```diff
@@ -52,3 +52,30 @@ module "catalogue_pipeline_2022-11-19" {
     aws.catalogue = aws.catalogue
   }
 }
+
+module "catalogue_pipeline_2022-11-28" {
```

Comment:
I have not created this pipeline or populated it or anything at all.

By including the extra pipeline in this file, it blocks deployment of the new code to the current production pipeline.

Depending on when I actually do it, I will add the correct date to both here and the API.